src/app/pages/fields/law/law-detail.component.ts

```diff
@@ -0,0 +1,42 @@
+import { Router } from '@angular/router';
+import { Country } from 'app/models/country.model';
+import { CountriesService } from 'app/services/countries.service';
+import { AuthService } from 'app/services/auth.service';
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'otp-law-detail',
+  templateUrl: './law-detail.component.html',
+  styleUrls: ['./law-detail.component.scss']
+})
+export class LawDetailComponent implements OnInit {
+
+  private countries: Country[];
+  private titleText: String = 'New Law';
```

Comment:
I was thinking of reusing the same component for editing and thus the text in the header will not always be the same